x-pack/plugins/cases/public/containers/configure/use_configure.tsx

```diff
@@ -202,6 +215,13 @@ export const useCaseConfigure = (): ReturnUseCaseConfigure => {
     });
   }, []);
 
+  const setID = useCallback((id: string) => {
```

Comment:
I do not think that we need the useCallback here

Reply:
You are right. `dispatch` should be memoized.

Reply:
So it would look like this instead?

```suggestion
  const setID = (id: string) => {
        dispatch({
          payload: id,
          type: 'setID',
        });
     };
```

Looks like we have a `useCallback` for each of these `set*` functions, should we refactor all of them or just the `setID`?